framework/Samplers/Sampler.py

```diff
@@ -298,7 +298,7 @@ def readSamplerInit(self,xmlNode):
         self.initSeed = Distributions.randomIntegers(0,2**31,self)
         for childChild in child:
           if childChild.tag == "limit":
-            self.limit = childChild.text
+            self.limit = int(childChild.text)
```

Comment:
I wholeheartedly agree with that idea.  Would it be too painful to add that as part of this MR?